js/form.js

```diff
@@ -0,0 +1,23 @@
+const adForm = document.querySelector('.ad-form');
+const mapFilter = document.querySelector('.map__filters');
+const adFormFieldsets = adForm.querySelectorAll('fieldset');
+const mapFilterFieldsets = mapFilter.querySelectorAll('fieldset');
+const mapFilterSelects = mapFilter.querySelectorAll('select');
+
+const getInactiveFormsStatus = () => {
+  adForm.classList.add('ad-form--disabled');
+  mapFilter.classList.add('map__filters--disabled');
+  adFormFieldsets.forEach((item) => {item.disabled = true;});
+  mapFilterSelects.forEach((item) => {item.disabled = true;});
+  mapFilterFieldsets.forEach((item) => {item.disabled = true;});
+};
+
+const getActiveFormsStatus = () => {
+  adForm.classList.remove('ad-form--disabled');
+  mapFilter.classList.remove('map__filters--disabled');
+  adFormFieldsets.forEach((item) => {item.disabled = false;});
```

Comment:
```suggestion
const toggleFormDisabled = (isDisabled) => {
  adForm.classList.toggle('ad-form--disabled', isDisabled);
  mapFilter.classList.toggle('map__filters--disabled', isDisabled);
  adFormFieldsets.forEach((item) => {item.disabled = isDisabled;});
```
в целом ты можешь иметь одну функцию, вместо двух